setup.py

```diff
@@ -70,6 +70,7 @@ def get_packages(package):
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
Also this seems to warrant an "Add official support for Python 3.9" PR and changelog entry?

Reply:
Good call, yup.